gov/governance/execute.gno

```diff
@@ -1,622 +1,311 @@
 package governance
 
 import (
+	"errors"
 	"std"
 	"strconv"
 	"strings"
 	"time"
 
+	"gno.land/p/demo/avl"
 	"gno.land/p/demo/ufmt"
-
 	"gno.land/r/gnoswap/v1/common"
-	"gno.land/r/gnoswap/v1/consts"
-
-	"gno.land/r/gnoswap/v1/gns"
 
-	cn "gno.land/r/gnoswap/v1/common"
 	cp "gno.land/r/gnoswap/v1/community_pool"
 	en "gno.land/r/gnoswap/v1/emission"
-	pl "gno.land/r/gnoswap/v1/pool"
-	pf "gno.land/r/gnoswap/v1/protocol_fee"
-	rr "gno.land/r/gnoswap/v1/router"
-	sr "gno.land/r/gnoswap/v1/staker"
 )
 
-// Execute executes the given proposal.
-// It checks various conditions such as voting period, execution window, quorum, and majority.
-// ref: https://docs.gnoswap.io/contracts/governance/execute.gno#execute
-func Execute(proposalId uint64) {
-	common.IsHalted()
-
-	en.MintAndDistributeGns()
-	updateProposalsState()
+var (
+	errInvalidMessageFormat = errors.New("invalid message format")
```

Comment:
make it in errors.gno with code.

Reply:
https://github.com/gnoswap-labs/gnoswap/pull/448/commits/eef01c3340a238102fd9f5e4063fd6da9b437581